packages/flutter_tools/lib/src/context_runner.dart

```diff
@@ -127,6 +131,12 @@ Future<T> runInContext<T>(
       FuchsiaDeviceTools: () => FuchsiaDeviceTools(),
       FuchsiaSdk: () => FuchsiaSdk(),
       FuchsiaWorkflow: () => FuchsiaWorkflow(),
+      GitHubTemplateCreator: () => GitHubTemplateCreator(
```

Comment:
Does this need to be in the context?  We're trying to move as much out of the context and globals as possible for testability, and g3 doesn't need to override it.

Can we inject fileSystem, flutterProjectFactory, and client into CrashReporter and let the Flutter version create a `GitHubTemplateCreator`, and the g3 version just ignore any params it doesn't need?

Then there's no need for a nullable `GitHubTemplateCreator` injected into `CrashReporter`